scripts/install_third_party_libs.py

```diff
@@ -220,7 +220,6 @@ def main() -> None:
     # looks for them in the default system paths when it is run. Therefore, we
     # must install these libraries to the developer's computer.
     system_pip_dependencies = [
-        ('enum34', common.ENUM_VERSION),
```

Comment:
Just to check, why was this deleted?

Reply:
We no longer need this, enum is now part of Python stdlib.